spacy/language.py

```diff
@@ -248,6 +248,18 @@ def pipe_names(self):
         """
         return [pipe_name for pipe_name, _ in self.pipeline]
 
+    @property
+    def pipe_labels(self):
+        """Get the labels set by the pipeline components, if available.
+
+        RETURNS (dict): Labels keyed by component name.
+        """
+        labels = {}
```

Comment:
We probably want this to be `OrderedDict` to be kind to Python2 and Python3.5
```suggestion
        labels = OrderedDict()
```

Reply:
Yeah, fair enough!